playground/frontend/playground_components/lib/src/repositories/models/run_code_request.dart

```diff
@@ -16,16 +16,19 @@
  * limitations under the License.
  */
 
+import '../../api/v1/api.pb.dart' show Dataset;
```

Comment:
Use `models/dataset.dart` here. Datasets should be converted to proto format at the same place where `Sdk` is.

Reply:
done